repo_files/documentations/line_chart.md

```diff
@@ -105,16 +105,39 @@ LineChart(
 |color|color of the line|Colors.black|
 |strokeWidth|strokeWidth of the line|2|
 
+### HorizontalRangeAnnotation
+|PropName|Description|default value|
+|:-------|:----------|:------------|
+|y1|start interval of horizontal rectangle|null|
+|y2|end interval of horizontal rectangle|null|
+|color|color of the rectangle|Colors.white|
+
+
+### VerticalRangeAnnotation
+|PropName|Description|default value|
+|:-------|:----------|:------------|
+|x1|start interval of vertical rectangle|null|
+|x2|end interval of vertical rectangle|null|
+|color|color of the rectangle|Colors.white|
+
 
 ### ExtraLinesData
 |PropName|Description|default value|
 |:-------|:----------|:------------|
+|extraLinesOnTop|determines to paint the extraLines over the trendline or below it|false|
```

Comment:
This is set `true` as the default value in the code, and for sure it should be true, change it to true.

Reply:
> This is set `true` as the default value in the code, and for sure it should be true, change it to true.

check it

Reply:
5417a8a

Reply:
not showing up in the this page (github bug) for some reason but its on the branch commit history

Reply:
Okay, thanks!